server/planning/events/events_spike.py

```diff
@@ -62,6 +62,52 @@ def update(self, id, updates, original):
 
         return item
 
+    def on_updated(self, updates, original):
+        # Spike associated planning
+        planning_spike_service = get_resource_service('planning_spike')
+        for planning in list(get_resource_service('planning').find(where={
```

Comment:
1. There is no need for `list(` here as your're already iterating over a cursor object
2. It would be better to use `search(` here with a source query to use elastic instead of mongo
3. There is no need to check `if planning['state'] == 'draft'` as you're already doing this in your query